src/app/treelist/treelist.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, Input, TemplateRef, ViewChild, ViewEncapsulation } from '@angular/core';
+import { TreeNode, TREE_ACTIONS, KEYS, IActionMapping, TreeComponent } from 'angular2-tree-component';
+
+// See docs: https://angular2-tree.readme.io/docs
+@Component({
+  selector: 'alm-tree-list',
+  templateUrl: './treelist.component.html',
```

Comment:
need to change this to template: require('./treelist.component.html'),

Same for styels. Take off "Url" and add require()